index.html

```diff
@@ -0,0 +1,113 @@
+            <li><a class="mobile-nav-i" href="about.html">About</a></li>
+            <li><a class="mobile-nav-i" href="#program">Program</a></li>
+            <li><a class="mobile-nav-i" href="#join">Join</a></li>
+            <li><a class="mobile-nav-i" href="#sponsor">Sponsor</a></li>
+            <li><a class="mobile-nav-i" href="#updates">News</a></li>
+            <li><a href="#" class="mt-4 p-3" id="cc-btn">CC Campaign</a></li>
+        </ul>
+    </nav>
+    <main class="top-section">
+        <section class="top-bg">
+            <div class="px-[10%] bg-[#5f5ff50] h-screen flex justify-center flex-col gap-5">
+                <div class="hero text-[#ec5242]">
+                    <h2 class="text-[25px] md:text-[50px]">"Hello! Sharing World"</h2>
+                    <h1 class="hero-first-title">Legends Of Sound<br>Festival</h1>
+                </div>
+                <div class="bg-[#d3d3d3] text-[#ec5242] w-[100%] md:w-[65%] px-[5%] py-[1%] border-solid border-2 border-[fff]">
+                    <p class="text-p p-3 border border-white">Grab your friends, grab your dancing shoes and let's go! Let's make this festival an unforgettable experience!</p>
+                </div>
+                <div class="text-[#272a31]">
+                    <h2 class="text-time text-[25px] md:text-[40px] text-[#272a31]">📆23 June(Fri) to 25 June(Sun) 2023</h2>
+                    <p>📍39 Gwi Gwi Mrwebi St, Newtown.</p>
+                </div>
+            </div>
+        </section>
+        <section class="program">
+            <br>
+            <h4 class="text-center section-title-white">Main Program</h4>
+            <hr class="line-name">
+            <div class="program-container"></div>
+            <br>
+            <div class="first-container text-center">
+                <a href="#" class="link-to-program p-3">See The Whole Program</a>
+            </div><br><br>
+        </section>
+        <section class="first-speaker">
+            <br>
+            <h4 class="text-center section-title-black">Featured Speakers</h4>
+            <hr class="line-name">
+            <div class="first-container"><div class="carry-speaker row"></div></div>
+            <div class="see-more rounded"><p class="text-center pt-2">More<i class="fa-sharp fa-solid fa-chevron-down more-icon"></i></p></div><br><br>
+        </section>
+        <section class="more-partners bg-secondary">
+            <div class="first-container"><br>
+                <h4 class="text-center section-title-white">Partner</h4><hr class="line-name">
+                <div class="first-container p-div text-center">
+                    <div class="first-group">
+                        <h4 class="first-p">Spotify</h4>
+                        <h4 class="second-p">YouTube</h4>
+                        <h4 class="third-p">DStv</h4>
+                        <h4 class="fourth-p">Amazon</h4>
+                        <h4 class="fifth-p">TikTok</h4>
+                    </div>
```

Comment:
- [ ] As i mentioned above, referring to the requirement of replicating the provided design, :+1:  I think this section needs an improvement, 
- -  Currently, the Parners are joined together and are not properly spaced like the one in the design, I'll suggest you look closy to the design and update this section. :+1: :100: 

- - Given Design:
![Screenshot from 2023-05-25 11-56-22](https://github.com/Kgomotso196/FirstCapstoneProject/assets/107632780/a341d46a-a9e2-4a52-aaa0-84ffd1ad3c3c)

- - Your Current design:
![Screenshot from 2023-05-25 11-55-57](https://github.com/Kgomotso196/FirstCapstoneProject/assets/107632780/050cd850-0c4e-4db9-a145-794544be9616)